### PR TITLE
82439 dev price information validation

### PR DESCRIPTION
**Pull request by sakshi15525**
1. Added validations for unit of sale price information.
2. Added unit test cases.
3. Moved Models and Services folder to common for reuse in price change webjob
4. Added FTs for Unit of sale price information

### CODE REVIEW

**Review by Shirin14926 — APPROVED** [review]
LGTM. Thanks

File: tests/UKHO.ERPFacade.API.FunctionalTests/ERPFacadeGeneratedJSONFiles/0807921f-df2b-4255-b6cf-3e196901b807.json
[the remark this answers is not in the document]
Comment (shrey1501620) [reply tests/UKHO.ERPFacade.API.FunctionalTests/ERPFacadeGeneratedJSONFiles/0807921f-df2b-4255-b6cf-3e196901b807.json:1]: done

**Review by Sadha1501493 — APPROVED** [review]

File: tests/UKHO.ERPFacade.API.FunctionalTests/Helpers/JSONHelper.cs [hunk 1]
```diff
@@ -0,0 +1,40 @@
+﻿using FluentAssertions;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
```
[the remark this answers is not in the document]
Comment (shrey1501620) [reply tests/UKHO.ERPFacade.API.FunctionalTests/Helpers/JSONHelper.cs]: done

**Review by RaviK2233 — COMMENTED** [review]
Reviewed FT and Looks good. @Sadha1501493 some of the checks have failed, could you please have a look, thanks

**Comment by Sadha1501493** [comment]
> Reviewed FT and Looks good. @Sadha1501493 some of the checks have failed, could you please have a look, thanks

@RaviK2233 : Checks are passing now. Please approve..

**Review by RaviK2233 — APPROVED** [review]
Looks good

**Review by jrippington — APPROVED** [review]
Looks good

**Review by DanOwenUKHO — APPROVED** [review]

**Pull request merged**